tests/integration/jit/tracing/test_span_events_are_created_upon_old_uncaught_handled_exception.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+/*DESCRIPTION
+Test that span events are correctly created from any eligible segment, even
+when an exception is handled by the exception handler.  In the case of PHP 8.0/8.1,
+PHP OAPI additionally where PHP OAPI additionally passes exception information in the
+zend_execute_data for the agent to use to create an error_event.
+Check that error events are created.
+*/
+
+/*SKIPIF
+<?php
+
+if (version_compare(PHP_VERSION, "8.2", ">=")) {
+  die("skip: PHP > 8.1 not supported\n");
+}
+//else {
+require('skipif.inc');
+//}
```

Comment:
Already handled in another change.

Order seemed to matter for the test runner on x64 8.2 instances.

Since it tried to run the test on gnu and musl , amd64, 8.2 which it should have skipped (and which oddly skipped okay the first time).

Going with this version:
```
if (version_compare(PHP_VERSION, "8.2", ">=")) {
  die("skip: PHP > 8.1 not supported\n");
}
require('skipif.inc');
```